app/decorators/service_decorator.rb

```diff
@@ -47,6 +47,18 @@ def api_selector_api_link
 
   alias link api_selector_api_link
 
+  def new_application_data
+    {
+      id: id.to_s,
+      name: name,
+      systemName: system_name,
+      updatedAt: updated_at,
+      appPlans: object.plans.stock.select(:id, :name).as_json(root: false),
```

Comment:
I think `object` is not necessary but it works too :)
You generally use `#object` when there is conflicting methods inside the decorator (like you defined method `#plans`